Generate .gitignore on init command (#2248)
MarkBind/markbind: docs/userGuide/gitignoreFile.md

.idea/*
*.iml
```

<box type="info">

> All pre-existing `.gitignore` file will not be overwritten if you are converting using the `markbind init --convert` command. 

Reviewer: ```suggestion
A pre-existing `.gitignore` file will not be overwritten if you are using the `markbind init --convert` command. 
```

Author: Updated in [6f09122](https://github.com/MarkBind/markbind/pull/2248/commits/6f091229ca498782376f4be82f5e6b3b1257c035)!
